setup_jbpfp_env.sh

```diff
@@ -2,4 +2,5 @@
 
 export JBPFP_PATH="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd)"
 export NANO_PB=$JBPFP_PATH/3p/nanopb
+export JBPFP_OUT_DIR=$JBPFP_PATH/out
```

Comment:
What are the outputs of the build beyond the cli tool? If the binary cli tool is the only one, perhaps we need to rename out to bin, to be more descriptive.

Reply:
This repo inherits the build from jbpf. In addition to jbpf-protobuf artifacts (the cli) the entire jbpf output is also in `$JBPFP_OUT_DIR`.

If you suggest to build it another way, let me know.